diff -ru greenbone-security-assistant-7.0.2/CMakeLists.txt greenbone-security-assistant-7.0.2.new/CMakeLists.txt
--- greenbone-security-assistant-7.0.2/CMakeLists.txt	2016-12-27 10:31:26.000000000 +0100
+++ greenbone-security-assistant-7.0.2.new/CMakeLists.txt	2017-04-23 23:50:33.262168626 +0200
@@ -232,7 +232,13 @@
 set (GSA_LOCALE_DIR "${GSA_DATA_DIR}/${GSA_LOCALE_SUBDIR}")
 set (GSA_CHROOT_LOCALE_DIR "/${GSA_LOCALE_SUBDIR}")
 
+set (OPENVAS_CERT_DIR     "${SYSCONFDIR}/pki/openvas")
+
 # TODO: Eventually use own certificates
+set (OPENVAS_SERVER_CERTIFICATE "${OPENVAS_CERT_DIR}/CA/servercert.pem")
+set (OPENVAS_SERVER_KEY         "${OPENVAS_CERT_DIR}/private/CA/serverkey.pem")
+set (OPENVAS_CA_CERTIFICATE     "${OPENVAS_CERT_DIR}/CA/cacert.pem")
+
 set (OPENVAS_SERVER_CERTIFICATE "${OPENVAS_STATE_DIR}/CA/servercert.pem")
 set (OPENVAS_SERVER_KEY         "${OPENVAS_STATE_DIR}/private/CA/serverkey.pem")
 set (OPENVAS_CA_CERTIFICATE     "${OPENVAS_STATE_DIR}/CA/cacert.pem")
